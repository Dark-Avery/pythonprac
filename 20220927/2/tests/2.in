1,2,10,125,1000