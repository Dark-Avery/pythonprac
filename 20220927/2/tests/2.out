[10, 1000, 1, 2, 125]
